test/e2e/components/vl-side-sheet.js

```diff
@@ -0,0 +1,26 @@
+const { VlElement } = require('vl-ui-core');
+const { By } = require('selenium-webdriver');
+
+class VlSideSheet extends VlElement {  
+    async _getSideSheet() {
+        return this.shadowRoot;
+    }
+
+    async _getCloseButton() {
+        return (await this._getSideSheet()).findElement(By.css('button.vl-side-sheet__close'));
+    }
+
+    async isOpen() {
+        return (await this._getSideSheet()).hasAttribute('open');
+    }
+
+    async close() {
+        return (await this._getCloseButton()).click();
+    }
+
+    async getContent() {
+        return (await this._getSideSheet()).getText();
+    }
```

Comment:
gade nie meestal den innerHTML hier van willen hebben ipv enkel den text?

Reply:
Ik heb het opgesplitst in een getText() en getContent(). De eerste geeft de text, de andere een VlElement waarvan je alle attributen etc kan opvragen.